docker/Dockerfile

```diff
@@ -54,7 +54,7 @@ RUN export ARCH="$(uname -m)" && if [[ ${ARCH} == "x86_64" ]]; then export ARCH=
 
 # Install openssl for FIPS support
 #@follow_tag(registry.redhat.io/ubi9/ubi-minimal:latest)
-FROM registry.access.redhat.com/ubi9-minimal:9.4-1134 AS runtime
+FROM registry.access.redhat.com/ubi9-minimal:9.4-1194@sha256:104cf11d890aeb7dd5728b7d7732e175a0e4018f1bb00d2faebcc8f6bf29bd52 AS runtime
```

Comment:
this doesn't work downstream in OSBS so we would need to remove the `@sha256:...` via the midstream code.

Reply:
should be OK after this: https://gitlab.cee.redhat.com/rhidp/rhdh/-/commit/cdf32ce02e98ef09ebde9cb0ffc009d9f9ccb17b